libglusterfs/src/client_t.c

```
@@ -598,7 +598,7 @@ gf_client_dump_fdtables(xlator_t *this)
                 clienttable->cliententries[count].next_free)
                 continue;
             client = clienttable->cliententries[count].client;
-            if (client->client_uid) {
+            if (client) {
```

At this point, isn't it fatal for client to be null? shouldn't we return an error here instead of proceeding?